README.md

```diff
@@ -20,7 +20,7 @@ AutoGluon automates machine learning tasks enabling you to easily achieve strong
 # python3 -m pip install --upgrade "mxnet<2.0.0"
 # python3 -m pip install autogluon
```

Comment:
will these still suffice for the example to run?

Reply:
should we somewhere add explicit announcement that this major change in APIs from v0.14?

Reply:
Adding to the rest of the modules

Reply:
We should be good. Just added setuptools upgrade as Nick suggested.